app/js/directives.js

```diff
@@ -1155,16 +1150,16 @@ angular.module('myApp.directives', ['myApp.filters'])
       }
 
       var transform = false
-      var trs = ['transform', 'webkitTransform', 'MozTransform', 'msTransform', 'OTransform'],
-        i
+      var trs = ['transform', 'webkitTransform', 'MozTransform', 'msTransform', 'OTransform']
+      var i
       for (i = 0; i < trs.length; i++) {
         if (trs[i] in historyMessagesEl.style) {
           transform = trs[i]
           break
         }
       }
 
-      var animated = transform && false ? true : false
+      var animated = transform && false ? true : false // ?
```

Comment:
I had a question about the assignment of this variable
As I'm reading this now, it will be equal to ``var animated = false``
what was the reason that it was done this way?